core/instrumentation.ts

```diff
@@ -26,3 +28,10 @@ export async function register() {
 		logger.info("NEXT_RUNTIME is not `nodejs`; skipping OTEL registration.");
 	}
 }
+
+export const onRequestError: InstrumentationOnRequestError = async (...args) => {
+	// necessary in order to avoid OOM errors on build due to Sentry making it
+	// hard for webpack to optimize
+	const onRequestError = await import("./instrumentation.onRequestError.ts");
+	return onRequestError.onRequestError(...args);
+};
```

Comment:
i did this dynamic import here as well because i thought it might help prevent some OOM errors during build. I don't think it does, but it's fine to keep it like so (although maybe i should remove the comment)